cli/tsc/99_main_compiler.js

```diff
@@ -4,7 +4,7 @@
 // that is created when Deno needs to compile TS/WASM to JS.
 //
 // It provides two functions that should be called by Rust:
-//  - `bootstrapCompilerRuntime`
+//  - `startup`
```

Comment:
Bike-shedding: I liked previous name better... 🤷

Reply:
well bootstraping isn't really what we are doing here ("a self-starting process that is supposed to proceed without external input"), and I am not sure why we need to indicate that it is the compiler runtime within the compiler runtime, if there were more functions like this, it might make sense to make them less ambiguous.